hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -71,6 +71,9 @@
   @Parameter(names = {"--use-jdbc"}, description = "Hive jdbc connect url")
   public Boolean useJdbc = true;
 
+  @Parameter(names = {"--enable-create-database"}, description = "Enable create hive database")
+  public Boolean enableCreateDatabase = false;
```

Comment:
rename: autoCreateDatabase ?

Reply:
Also should the value be `true` by default to preserve existing behavior

Reply:
@vinothchandar We do not let hudi create databases by default. So false seems to be ok :) @bvaradar to chime in here.

Reply:
https://lists.apache.org/thread.html/e1b7f97c774e1d7d7fc54fbb46db49aaf2e217303a50d9885150242d%40%3Cdev.hudi.apache.org%3E - this is what I am referring to. :)

Reply:
Ok, my bad. Missed changes for creation of hive database.